core/src/main/java/com/airbnb/aerosolve/core/features/Features.java

```diff
@@ -4,16 +4,18 @@
 import com.airbnb.aerosolve.core.FeatureVector;
 import com.airbnb.aerosolve.core.util.Util;
 import com.google.common.annotations.VisibleForTesting;
-import lombok.experimental.Builder;
-import lombok.extern.slf4j.Slf4j;
 import org.apache.commons.lang3.tuple.ImmutablePair;
 import org.apache.commons.lang3.tuple.Pair;
 
 import java.lang.reflect.Field;
 import java.util.*;
 
-@Builder @Slf4j
 public class Features {
+  public final static String DEFAULT_STRING_FAMILY = "L";
```

Comment:
But could this have unintended consequences if people didn't realize this was 'L' and had another family of the same name coincidentally?